samples/03_org_administrators/org_wide_dependency_graph.ipynb

```diff
@@ -0,0 +1,1264 @@
+{
```

Comment:
<p>Replace <em>Sorting of org items </em>with <em>Sorting of items in our organization</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2311/discussion/#comment-2193902648'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>